.pre-commit-config.yaml

```diff
@@ -7,48 +7,84 @@ repos:
+        stages: [commit, push]
     -   id: debug-statements            # Check for debugger imports and py37+ breakpoint() calls in python source.
+        stages: [commit, push]
     # -   id: detect-aws-credentials      # Checks for the existence of AWS secrets that you have set up with the AWS CLI
-    # -   id: detect-private-key          # Checks for the existence of private keys.
+    #     stages: [commit, push]
+    -   id: detect-private-key          # Checks for the existence of private keys.
+        stages: [commit, push]
     # -   id: double-quote-string-fixer   # This hook replaces double quoted strings with single quoted strings.
+    #     stages: [commit, push]
     -   id: end-of-file-fixer           # Makes sure files end in a newline and only a newline.
+        stages: [commit, push]
     # -   id: fix-encoding-pragma         # Add # -*- coding: utf-8 -*- to the top of python files.
+    #     stages: [commit, push]
     # -   id: file-contents-sorter        # Sort the lines in specified files (defaults to alphabetical). You must provide list of target files as input to it.
+    #     stages: [commit, push]
     # -   id: flake8                      # Run flake8 on your python files.
+    #     stages: [commit, push]
     -   id: forbid-new-submodules       # Prevent addition of new git submodules.
-    # -   id: mixed-line-ending           # Replaces or checks mixed line ending.
+        stages: [commit, push]
+    -   id: mixed-line-ending           # Replaces or checks mixed line ending.
+        stages: [commit, push]
     # -   id: name-tests-test             # Assert that files in tests/ end in _test.py
+    #     stages: [commit, push]
     # -   id: no-commit-to-branch         # Protect specific branches from direct checkins.
+    #     stages: [commit, push]
     # -   id: pyflakes                    # Run pyflakes on your python files.
-    # -   id: pretty-format-json          # Checks that all your JSON files have keys that are sorted and indented.
+    #     stages: [commit, push]
+    -   id: pretty-format-json          # Checks that all your JSON files have keys that are sorted and indented.
+        stages: [commit, push]
     # -   id: requirements-txt-fixer      # Sorts entries in requirements.txt, removes 0.0.0 entries
+    #     stages: [commit, push]
     # -   id: sort-simple-yaml            # Sorts simple YAML files which consist only of top-level keys, preserving comments and blocks.
+    #     stages: [commit, push]
     -   id: trailing-whitespace         # Trims trailing whitespace.
+        stages: [commit, push]
+
 -   repo: https://github.com/pre-commit/mirrors-yapf
     rev: v0.22.0
     hooks:
     -   id: yapf
-        args: ['-dpr']
+        stages: [commit, push]
+
 -   repo: https://github.com/pre-commit/mirrors-pylint
     rev: v1.9.1
+    stages: [commit, push]
     hooks:
     -   id: pylint
-        args: ['--rcfile=setup.cfg']
```

Comment:
I think you need to keep args

Reply:
Or else pre-commit will say `No files to check(skipped)`

Reply:
Oops not sure how these got removed